### PR TITLE
Shivangi/node manager 5774

### PR DESCRIPTION
**Pull request by shivangi-singh1**
### :nut_and_bolt: Description: What code changed, and why?
1.Allowed searching Node Managers by Name.
2.Allowed filtering Node Managers by Type.(Azure, AWS, GCP)
3.Allowed load more option on Node Managers.

### :chains: Related Resources 
UI Story to allow more than Node Manager listing #5774 

### :+1: Definition of Done
1. Allow searching Node Managers by Name.
2.Allow filtering Node Managers by Type.(Azure, AWS, GCP)
3.Allow load more Node Managers.

### :athletic_shoe: How to Build and Test the Change

```
STEP 1
inside the hab studio

[default:/src:0]# build components/automate-ui-devproxy/
[default:/src:0]# start_automate_ui_background
[default:/src:0]# start_all_services

STEP 2
open new window
go to automate UI path

$ cd components/automate-ui
and run the command 

npm run serve:hab
```

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [x] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [x] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [x] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [x] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable

https://user-images.githubusercontent.com/87964433/140722628-f0cc39cd-8263-408e-91a1-3908e292f00c.mov

### CODE REVIEW

File: components/automate-ui/src/app/page-components/job-nodes-form/job-nodes-form.component.html [hunk 1]
```diff
@@ -1,12 +1,53 @@
+
 <div [formGroup]="form">
   <h3>Configure your custom parameters to add nodes</h3>
+  <div class="nodeOptions">
+      <form (ngSubmit)="onSearchInput(search_box.value)" style="display: flex;     width: 46%;">
```
Comment (vivekshankar1) [inline components/automate-ui/src/app/page-components/job-nodes-form/job-nodes-form.component.html:5]: Please move styles to scss file

Comment (shivangi-singh1) [reply components/automate-ui/src/app/page-components/job-nodes-form/job-nodes-form.component.html]: done

**Comment by shaik80** [comment]
can you please resolve the conflict

**Comment by shivangi-singh1** [comment]
> can you please resolve the conflict

Done

**Comment by shaik80** [comment]
please make sure to resolve pipeline failure

**Review by vivekshankar1 — APPROVED** [review]
Looks good to me.

**Review by vivek-yadav — APPROVED** [review]

File: .expeditor/verify.pipeline.yml [hunk 2]
```diff
@@ -76,7 +76,7 @@ steps:
     expeditor:
       executor:
         docker:
-          
+
```
Comment (vivek-yadav) [inline .expeditor/verify.pipeline.yml:79]: why is this added
